feed.xml

```diff
@@ -1,30 +0,0 @@
----
```

Comment:
Yup! It generates the file as part of the build process.

Link to the gem here, if you're curious: https://github.com/jekyll/jekyll-feed